index.html

```diff
@@ -230,5 +230,38 @@ <h3 class="education-title">Skills</h3>
         </div>
         </section>
     </section>
+
+
+        <!-- Footer and contact form -->
+
+
+        <footersection class="form">
+            <div class="Contact-form-description">
+                <h1 class="Contact-form-description">
+                    I'm always interested in hearing
+                    about new projects, so if you'd
+                    like to chat please get in touch.
+                </h1>
```

Comment:
- [OPTIONAL]  I suggest removing the `<h1>` tag from here and adding `<p>` tag or another hx but It would be better to use `<p>` tag to Follow best practices for HTML 👌 .
> **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).